core/services/chainlink/testdata/config-full.toml

```
@@ -0,0 +1,275 @@
+Dev = true
+ExplorerURL = 'http://explorer.url'
+InsecureFastScrypt = true
+ReaperExpiration = '168h0m0s'
+RootDir = 'test/root/dir'
+ShutdownGracePeriod = '10s'
+FeatureFeedsManager = true
+FeatureUICSAKeys = true
+FeatureLogPoller = true
+FMDefaultTransactionQueueDepth = 100
+FMSimulateTransactions = true
+FeatureOffchainReporting2 = true
+FeatureOffchainReporting = true
+
+[Database]
+ListenerMaxReconnectDuration = '1m0s'
+ListenerMinReconnectInterval = '5m0s'
+Migrate = true
+ORMMaxIdleConns = 7
+ORMMaxOpenConns = 13
+TriggerFallbackDBPollInterval = '2m0s'
+AdvisoryLockCheckInterval = '5m0s'
+AdvisoryLockID = 345982730592843
+LockingMode = 'advisory'
+BackupDir = 'test/backup/dir'
+BackupOnVersionUpgrade = true
+BackupURL = 'http://test.back.up/fake'
```

Same option here:
```suggestion
[Database.Backup]
Dir = 'test/backup/dir' 
OnVersionUpgrade = true
URL = 'http://test.back.up/fake'
Freqeuncy = '1h'
```
FWIW these groupings would align very nicely with docs.chain.link

---

Done